jekyll/_docs/configuration.md

```diff
@@ -291,6 +291,9 @@ You can enable other databases and services from the `services` section:
 ```
 machine:
   services:
+    - mysql
+    - mongodb
+    - postgresql
```

Comment:
Thought that might only apply to Ubuntu 12.04 (old default) but it applies to 14.04 also. :/